carts/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Cart(models.Model):
+    user     = models.ForeignKey("users.User", on_delete=models.CASCADE, related_name="cart")
+    product  = models.ForeignKey("products.Product", on_delete=models.CASCADE, related_name="cart")
+    quantity = models.IntegerField()
```

Comment:
`default = 0` 옵션 추가해주시면 좋을 것 같습니다~
```suggestion
    quantity = models.IntegerField(default=0)
```